InteropTest.cs

```diff
@@ -24,6 +28,33 @@ internal class InteropTest
         public static excel.Application excelApp;
         public static powerPoint.Application powerPointApp;
 
+        /*
+         * Sometimes app.Quit() does not actually quit the application, or other instances of the application
+         * can be open, especially if a previous file errored or was interrupted. The open applications can
+         * cause issues when testing the next file. This force quits all instances by process name so that
+         * the next test file can be tested on a new instance of the application
+         */
+        public static void forceQuitAllApplication(string application)
```

Comment:
are we sure we are not going to be running multiple instances of tests? in that case it might cause problem.

Reply:
or i.e one process timed out and we started another one before killing it properly...

Reply:
There is definitely no need to run multiple instances testing the same application

I could add a check to stop multiple instances from running, and that would be easier than trying to figure out why MSO fails to close sometimes.

But I am going to do that later because while working on this I've been running multiple instances at the same time for different applications

Reply:
yup, makes sense.